.eslintrc.json

```diff
@@ -34,6 +34,7 @@
     "no-unused-vars": ["error", { "varsIgnorePattern": "^_", "argsIgnorePattern": "^_" }],
     "prefer-const" : "error",
     "no-eval": "error",
+    "max-len": ["error", { "code": 150 }],
```

Comment:
@devalevenkatesh @xuesichao Do we need to agree on this value :p

Reply:
Hmm, can make it 125 but I did not find any standard around this 😅 
https://hilton.org.uk/blog/source-code-line-length

Reply:
So one test locally, I think 125 is good for wide screen monitor and 80 is good for normal 27inch monitor. The 80 limit should be good when not using two windows side by side in VSCode for 27 inch one and 125 on a wide screen 34inch one.

According to above article GitHub's code length is 125, my comparison for this review takes about 102 characters 😅 .

Considering all I think 150 is on higher side in all cases.

![Screen Shot 2021-12-30 at 11 11 38 AM](https://user-images.githubusercontent.com/61809205/147781777-a27a6b92-f365-4fa7-830a-9c279579f142.png)

So, if we take side-by-side window: length should be 80. (Good for wide + small monitors + terminals + side stacked code reviews)

If we take no side-by-side considerations: 120 or 125 is preferred.

Reply:
@ltrung @xuesichao what do you think :) ?

Reply:
![image (1)](https://user-images.githubusercontent.com/54328451/147787894-c95bef5a-562b-43ab-9790-1946843e5c65.png)
![image](https://user-images.githubusercontent.com/54328451/147787895-50706bd2-7896-4de1-94dc-33ec6b3dcc0f.png)

I chose 150 based on this example. We can have everything on one line or separate lines - purely preference.

Reply:
Same code, just different line length max

Reply:
Going with 120 max